.eslintrc

```diff
@@ -4,6 +4,7 @@
     "max-len": ["error", 90, 4, {
       "ignoreComments": true,
       "ignoreUrls": true,
+      "ignoreStrings": true,
```

Comment:
just curious.. why we do need to add this option?

Reply:
So the linter doesn't complain about [this line](https://github.com/strongloop/loopback-connector/pull/149/files#diff-c7e079ebe967e7091d71564875378cbdR636) and [this line](https://github.com/strongloop/loopback-connector/pull/149/files#diff-7dbca0b5863a379be24f48aefaac8299R476):

`"ignoreStrings": true ignores lines that contain a double-quoted or single-quoted string`

Reply:
I see. I think in some other places, we break it into multiple lines using `+`, e.g. https://github.com/strongloop/generator-loopback/blob/master/datasource/index.js#L332.  
I'm good with either way.